lib/kredis/types/counter.rb

```diff
@@ -24,4 +20,9 @@ def value
   def reset
     del
   end
+
+  private
+    def set_default
+      set(default.to_i, ex: expires_in, nx: true) unless exists?
```

Comment:
Why do we need `unless exists?` if we're already using `nx: true`?

Reply:
Good catch.  Updated.